.vscode/launch.json

```diff
@@ -132,6 +132,17 @@
             "cwd": "${workspaceFolder}/vscode-lean4/out/",
             "outFiles": ["${workspaceFolder}/vscode-lean4/out/test/suite/**/*.js"],
             "preLaunchTask": "watchTest"
-        }
+        },
+        {
+            "name": "Launch runTests.js",
```

Comment:
Did you mean to add this launch config?

Reply:
yes, turns out it is handy to be able to debug this file also.